resources/nodogsplash.conf

```diff
@@ -401,17 +401,16 @@ FirewallRuleSet users-to-router {
 #
 # BinAuth /bin/myauth.sh
 
-# Nodogsplash uses specific values to mark packets using iptables.
-# Parameter: FW_MARK_BLOCKED
-# Default: 0x100
+# Nodogsplash uses specific HEXADECIMAL values to mark packets used by iptables as a bitwise mask.
+# This mask can conflict with the requirements of other packages such as mwan3, sqm etc
+# Any values set here are interpreted as in hex format.
 #
-# Parameter: FW_MARK_TRUSTED
-# Default: 0x200
+# Parameter: fw_mark_authenticated
+# Default: 30000 (0011|0000|0000|0000|0000 binary)
 #
-# Parameter: FW_MARK_AUTHENTICATED
-# Default: 0x400
+# Parameter: fw_mark_trusted
+# Default: 20000 (0010|0000|0000|0000|0000 binary)
```

Comment:
This has nothing to do with fas, can you put it into a different PR?

Reply:
I would prefer to keep it here as the original reason for adding to config some months or so ago was to use mwan3 and sqm scripts with NDS and enable a FAS to take advantage of and  manipulate them. The same goes for binauth.
I do see your argument, but it hardly matters as soon as we merge this.